### ISSUE TITLE
Bug: WinUI-2.5: Accessibility: [MAS4.2.1]: Narrator: The name property is not defined for the progress edit box

### ISSUE THREAD

**Issue by v-amnat** [microsoft/microsoft-ui-xaml#3634]
**Tags**
Accessibility, A11yFT,  A11yMAS, A11yMediumImpact, MAS4.2.1, Narrator, Sev3, WinUI-2.5, Win32, Wipro 

**User Impact**
The screen reader user would face difficulty to find the purpose of the edit box, if name property is null for the progress edit box

**Steps to reproduce the bug**
Launch the WinUI_2.5_TestApp
1. Go to 'Progress Spinner'  edit box
2. Run the Accessibility Insight for Windows for the Progress spinner.
3. Observe the narrator announcement for the edit box

**Expected behavior**
1. The name property should be defined for the progress edit box.
The should read the name of the edit box.

**Actual behavior**
The name property is not defined for the progress edit box, as name of the edit box is null/empty.

**Repro attachment**
[Narrator- Name of progress edit.zip](https://github.com/microsoft/microsoft-ui-xaml/files/5558843/Narrator-.Name.of.progress.edit.zip)

![image](https://user-images.githubusercontent.com/74585893/99507244-23b1d180-29a9-11eb-91bb-a6f500c0bd9c.png)

**Version Info**
Win32
Windows OS Version: Windows 10
App: WinUI_2.5_TestApp

**Comment by YuliKl** [comment]
Sounds like a dupe of #1854, which claims to be fixed but the issue is still reproing. Same problem mentioned in #2951.

**Comment by ranjeshj** [comment]
@teaP  can you take a look ?

**Comment by ranjeshj** [comment]
This is a dup of https://github.com/microsoft/Xaml-Controls-Gallery/issues/325
